### ISSUE TITLE
X-Palm-Api-Key error on non text2vec-palm vectorizer class

### ISSUE THREAD

**Issue by eostis** [weaviate/weaviate#3040]
`remote client vectorize: Palm API Key: no api key found neither in request header: X-Palm-Api-Key nor in environment variable under PALM_APIKEY`

This error is raised on a class with the OpenAI vectorizer, when 'text2vec-palm' is enabled (Weaviate v1.19.4)

```
replicas: 2

env:
  AUTOSCHEMA_ENABLED: false

modules:
  text2vec-openai:
    enabled: true

  text2vec-cohere:
    enabled: true

  multi2vec-clip:
    enabled: true

  qna-openai:
    enabled: true

  text2vec-palm:
    enabled: true


```

**Comment by eostis** [comment]
Solved with #3021

**Comment by antas-marcin** [comment]
Nice! glad to hear that it works!
